mock/ecs_client.go

```diff
@@ -122,19 +122,20 @@ type ECSCluster map[string]ECSTask
 
 // ECSTask represents a mock running ECS task within a cluster.
 type ECSTask struct {
-	ARN         *string
-	TaskDef     ECSTaskDefinition
-	Cluster     *string
-	Containers  []ECSContainer
-	Group       *string
-	ExecEnabled *bool
-	Status      *string
-	GoalStatus  *string
-	Created     *time.Time
-	StopCode    *string
-	StopReason  *string
-	Stopped     *time.Time
-	Tags        map[string]string
+	ARN               *string
+	TaskDef           ECSTaskDefinition
+	Cluster           *string
+	ContainerInstance *string
```

Comment:
This is just for testing purposes so that we can simulate a fake ECS task running in a parent container instance that's going to be spot instance interrupted. SNS sends us notifications for when the container instance is going to be interrupted, but it doesn't notify us when the individual ECS tasks in that container instance are going to be interrupted. Adding this field lets us associate the two so that when we call `ListTasks` with the soon-to-be-interrupted container instance, it tells us which tasks are running in it.